tests/fixtures/scenarios/packer/components/packer/aws/multi-file/README.md

```
@@ -0,0 +1,34 @@
+# Multi-File Packer Component Test
+
+This component tests Atmos support for directory-based Packer templates.
+
+## Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)